cmd/mountlib/mount.go

```diff
@@ -154,6 +154,21 @@ to use Type=notify. In this case the service will enter the started state
 after the mountpoint has been successfully set up.
 Units having the rclone ` + commandName + ` service specified as a requirement
 will see all files and folders immediately in this mode.
+
+### chunked reading ###
+
+--vfs-read-chunk-size will enable reading the source objects in parts.
+This can reduce the used download quota for some remotes by requesting only chunks
+from the remote that are actually read at the cost of an increased number of requests.
```

Comment:
You should probably mention here that it only works for --vfs-cache-mode < full

Reply:
This is a good idea.
I added notice to the docs in the latest commit.